### PR TITLE
[PAY-250] Implement SendTipScreen

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

- Implements SendTipScreen
- Adds error option for Text color
- Improves TextInput style propagation
- Fixes bn.js types
- Adds disabled styles to Button

https://user-images.githubusercontent.com/8230000/170389616-6da4a159-9885-499d-b6a7-f9979ec0ce20.mov

# Dragons
Still need to add the `tip x amount to show up on leaderboard` but wanted to get core happy path done first

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-250-tip-flow-happy-path

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-250-tip-flow-happy-path

File: packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+
 export const SendTipScreen = () => {
+  const styles = useStyles()
+  const [tipAmount, setTipAmount] = useState('')
+  const accountBalance = useSelectorWeb(getAccountBalance)
   const navigation = useNavigation<TipArtistNavigationParamList>()
 
+  const hasInsufficientBalance =
+    accountBalance &&
+    parseAudioInputToWei(tipAmount as StringAudio)?.gte(accountBalance)
```
Comment (sddioulde) [inline packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx:42]: should it just be greater than, aka `gt` instead of `gte`?

**Review by sddioulde — APPROVED** [review]
looks great. approving,  but i think the insufficient check should be `gt`

File: packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx (hunk 1 above)
Comment (piazzatron) [reply packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx:42]: also is there a case where we have no accountBalance? I'm guessing not, but maybe `!accountBalance || parse...` instead?

**Review by piazzatron — APPROVED** [review]

**Pull request merged**